PyInstaller/lib/modulegraph/modulegraph.py

```diff
@@ -2581,14 +2581,11 @@ def _scan_bytecode(
               parsing will have already parsed import statements, which this
               parsing must avoid repeating.
         """
-        constants = module_code_object.co_consts
-
         level = None
         fromlist = None
 
-        prev_insts = []
-
-        for inst in get_instructions(module_code_object):
+        prev_insts = deque(maxlen=2)
```

Comment:
I suggest adding a small comment about deque, since they are not that comment (I did not know them, too). Somethink like "deque is a list-like container with fast appends, pops on either end and discarding elements too much"